.ci/jobs.t/defaults.yml

```diff
@@ -0,0 +1,92 @@
+---
+
+##### GLOBAL METADATA
+
+- meta:
+    cluster: elasticsearch-ci
+
+##### JOB DEFAULTS
+
+- job:
+    vault:
+      url: https://secrets.elastic.co:8200
+      role_id: 1ba1ac3e-aee4-d040-d9a3-6ae23bd2b3db
+    node: "general-purpose"
+    concurrent: true
+    logrotate:
+      daysToKeep: 30
+      numToKeep: 90
+      artifactDaysToKeep: 7
+    scm:
+      - git:
+          name: origin
+          # master node jenkins user ~/.ssh
+          credentials-id: f6c7695a-671e-4f4f-a331-acdce44ff9ba
+          reference-repo: "/var/lib/jenkins/.git-references/elasticsearch.git"
+          branches:
+            - "%BRANCH%"
+          url: "https://github.com/elastic/elasticsearch.git"
+          basedir: ""
+          wipe-workspace: "yes"
+    wrappers:
+      - timeout:
+          type: absolute
+          timeout: "120"
+          fail: true
+      - ansicolor
+      - timestamps
+      # TODO: No support un JJBB ?
+      # - gradle-build-scan
+    properties:
+        - github:
+            url: https://github.com/elastic/elasticsearch/
+        - inject:
+            properties-content: |
+              HOME=$JENKINS_HOME
+    builders:
+       - inject:
+          properties-file: '.ci/java-versions.properties'
+          properties-content: |
+            JAVA_HOME=$HOME/.java/$ES_BUILD_JAVA
+            RUNTIME_JAVA_HOME=$HOME/.java/$ES_RUNTIME_JAVA
+            JAVA7_HOME=$HOME/.java/java7
+            JAVA8_HOME=$HOME/.java/java8
+            JAVA9_HOME=$HOME/.java/java9
+            JAVA10_HOME=$HOME/.java/java10
+            JAVA11_HOME=$HOME/.java/java11
+            JAVA12_HOME=$HOME/.java/openjdk12
+            JAVA13_HOME=$HOME/.java/openjdk13
+       - shell: |
+          #!/usr/local/bin/runbld --redirect-stderr {runbld_args}
```

Comment:
Does JJB support parameterization? What does `{runbld_args}` evaluate to here? I suspect this is just copy/paste leftovers from the original builder macro.

Reply:
you are correct, I changed it to default to last good  commit. 
We could also drop the `shell` part to have no default, maybe that would be less confusing

Reply:
👍 I think if the expectation is that the `shell` bit is always overridden then having a default might not make sense.